bin/runtests

```diff
@@ -52,7 +52,16 @@ DB_NAME_TEST=${DB_NAME}_test
 
 
 echo "Create database"
-createdb -O $DB_USER ${DB_NAME_TEST}
+if [[ ! -z "$DB_TEST_TEMPLATE" ]];
```

Comment:
Cool :) can you add a note in the readme?

Reply:
done